scripts/install_idyntree_bindings.m

```diff
@@ -0,0 +1,118 @@
+function install_idyntree_bindings(varargin)
+% authored by Silvio Traversaro
+% modified by Prashanth Ramadoss
+% install iDynTree matlab bindings in the local directory
+% setup a default install path
+% or allow the user to specify path as
+% install_iDynTree('installPrefix', '<path-to-install>')
+p = inputParser;
+default_install_prefix = fullfile(pwd, '../deps/', 'idyntree-matlab');
+p.addOptional('installPrefix', default_install_prefix);
+p.parse(varargin{:});
+
+install_prefix = p.Results.installPrefix;
+
+% If not on Windows, check if the install path contains a space as this
+% creates an installation error later (see
+% https://github.com/robotology/robotology-superbuild/issues/780)
+if ~ispc
+    if contains(strip(install_prefix), ' ')
+        fprintf('Install directory path %s contains a space.\n', install_prefix)
+        fprintf('Please use install_idyntree_bindings function in a directory path that does not have spaces.\n');
+        return;
+    end
+end
+
+setup_script = fullfile(pwd, 'idyntree_bindings_setup.m');
+
+if exist(install_prefix)
+    fprintf('Directory %s already present.\n', install_prefix);
+    fprintf('Please use the existing installation or delete the existing installation to proceed with the install.\n');
+    return;
+end
+
+fprintf('Installing iDynTree Matlab bindings in %s\n', install_prefix);
+
+% The install url is created following
+mambaforge_url_prefix = 'https://github.com/conda-forge/miniforge/releases/latest/download/';
```

Comment:
Differently that from reusable software (such as robotology-superbuild) for reproducible installation process, it is not a good idea to use `latest` as it could break in the future. Better to put the version that you tested:
```suggestion
mambaforge_url_prefix = 'https://github.com/conda-forge/miniforge/releases/4.11.0-0/download/';
```

Reply:
Thanks for the suggestion. I noticed the link has the following format,
```
https://github.com/conda-forge/miniforge/releases/download/4.11.0-0/
```
in https://github.com/conda-forge/miniforge/releases/tag/4.11.0-0, where the `download` precedes the version.

Reply:
Thanks for noticing, I did the suggestion without checking the actual format of the URL.

Reply:
Fixed in https://github.com/ami-iit/paper_ramadoss-2022-ral-humanoid-base-estimation/commit/2295787f336f3825bc6366453dbd792b52198801